.github/workflows/conftest.yaml

```
@@ -0,0 +1,43 @@
+name: Tier 1 - Manifest policy check
+
+on: [push, pull_request]
+
+jobs:
+  conftest-rbac:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)